### PR TITLE
refactor: change from input to textarea

### PR DESCRIPTION
**Pull request by schmelto**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -3,19 +3,19 @@ description: Create a bug report to help us address errors in the repository
...
 body:
-  - type: input
+  - type: textarea
     id: bugdescription
     attributes:
       label: Description of the bug
     validations:
       required: true
-  - type: input
+  - type: textarea
```
Comment (Panquesito7) [inline .github/ISSUE_TEMPLATE/bug.yml:12]: Hmm, this should probably be `input` as it is just a one-line text as seen in the example. 🙂

**Review by Panquesito7 — COMMENTED** [review]
Overall, it looks good. Great catch, @schmelto! 😄👍

File: .github/ISSUE_TEMPLATE/bug.yml (hunk 1 above)
Comment (schmelto) [reply .github/ISSUE_TEMPLATE/bug.yml:12]: I would leave it as textarea for me description of the bug includes more than one line of comment :)

Comment (Panquesito7) [reply .github/ISSUE_TEMPLATE/bug.yml:12]: This is not for the bug description, it is for the `Project stack or language` part. 🙂

**Comment by schmelto** [comment]
@Panquesito7 can you take a look again

**Review by Panquesito7 — APPROVED** [review]
LGTM. Thank you for improving the issue form, @schmelto. 😄👍

**Comment by dth99** [comment]
Add Profile Views in README.md

**Comment by dth99** [comment]
> Add Profile Views in README.md



> _No description provided._

Add Profile Views in README.md

**Review by Vyvy-vi — APPROVED** [review]
LGTM! Thanks for opening this PR.

**Comment by dth99** [comment]
> LGTM! Thanks for opening this PR.

Welcome

**Comment by schmelto** [comment]
> Add Profile Views in README.md


What do you mean by that?

**Comment by dth99** [comment]
> > Add Profile Views in README.md
> 
> What do you mean by that?

 I had made a PR to your's readme. :) hope you liked it

**Pull request merged**